src/dipdup/context.py

```diff
@@ -745,6 +745,9 @@ def __init__(
         )
         self.logger = logger
         self.hook_config = hook_config
+    
+    def __repr__(self) -> str:
+        return f'<Class {self.__class__.__name__}(logger={self.logger}, hook_config={self.hook_config})>'
```

Comment:
Please, leave only the hook name here.

Reply:
@droserasprout To get the hook name would be self.template_values._index?

Reply:
@droserasprout To get the hook name here how can I recover it from the self.hook_config? self.hook_config.name?

Reply:
Yes.